src/classes/_typography.scss

```diff
@@ -0,0 +1,25 @@
+$typographyClasses: (
+    "h1": ("bold", "normal", "light"),
```

Comment:
I think it can be cool to refactor the way we creating tokens to have only one config file and creating all  classes and tokens from it without additional files (right now we need both map file and sass variables file for creating only css variables)
But I think its out of current scope so I would love to add it to backlog